files/build_templates/docker_image_ctl.j2

```diff
@@ -126,12 +126,17 @@ start() {
 {%- if '--log-driver=json-file' in docker_image_run_opt or '--log-driver' not in docker_image_run_opt %}
         --log-opt max-size=2M --log-opt max-file=5 \
 {%- endif %}
-{%- if docker_container_name == "syncd" and sonic_asic_platform == "mellanox" %}
+{%- if sonic_asic_platform == "mellanox" %}
```

Comment:
what is the new logic? we dont need it as and? maybe you should comment the flow required and what sync is required

Reply:
@liat-grozovik these changes are done to fix review comments. It is simple rearrangement to avoid double clauses.